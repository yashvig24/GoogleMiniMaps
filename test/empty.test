# Tests build emptyrGraph.

# Load empty.tsv
LoadGraph empty_graph empty.tsv

# Print the nodes in the graph
ListNodes empty_graph