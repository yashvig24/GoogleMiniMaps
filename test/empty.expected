# Tests build emptyrGraph.

# Load empty.tsv
loaded graph empty_graph

# Print the nodes in the graph
empty_graph contains: